app/views/users/show.html.erb

```diff
@@ -0,0 +1 @@
+ <h1>all list of users will display here</h1>
```

Comment:
- [ ] [optional] Kindly change the heading of this page to ensure the user can distinguish it from the index page